### PR TITLE
Read account using Nylas Node.js SDK

### LINKED ISSUE #23: Read Account details using Node
Code sample to read account details using Nylas Node SDK.

### PR DESCRIPTION
**Pull request by amit-ksh**
Closes #23 

## Guidelines
* [x] PR follows the [Contributing guidelines](https://github.com/nylas-samples/nylas-hacktoberfest-2022/blob/main/CONTRIBUTION.md)?
* [x] PR Adheres to our [Code of Conduct](https://github.com/nylas-samples/nylas-hacktoberfest-2022/blob/main/CODE_OF_CONDUCT.md)?
* [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?
* [x] Ensure you are assigned the issue before creating a PR.
* [x] Only one issue per PR is allowed. This simplifies the code review.

## Code complete
* [x] A `README.md` available describing how to setup and test the code?
* [x] The code sample in the correct location (i.e. `code-sample/email/node/read-email`)?
* [x] Link added to the repository readme (view `# Code Snippets created by the Nylas Community` with examples) describing what the code sample does for others to try out
* [x] The code can be run using Github Codespaces, so the maintainer can try out the code sample remotely to ensure everything is working before we accept the PR.

## For PR Reviewer use
* [x] Purpose of code sample is clear
* [x] The code sample is readable
* [x] Steps try out the code are provided
* [x] Code sample runs on codespaces environment
* [x] Code works as expected
* [x] PR labels added

### CODE REVIEW

File: code-samples/account/node/read-account/.gitignore [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+.grunt
+_SpecRunner.html
+*.swp
+*.swo
+tags
+CTAGS
+.idea
+.env
+package-lock.json
+yarn.lock
```
Comment (relaxedtomato) [inline code-samples/account/node/read-account/.gitignore:15]: Great you added a `.gitignore`. Let's keep this in - we can always lean on the parent folder `.gitignore` as well.

File: readme.md [hunk 2]
```diff
@@ -82,6 +82,12 @@ Check out our [FAQs](https://github.com/nylas-samples/nylas-hacktoberfest-2022/b
 - [Send Emails with Rust and C#](https://github.com/nylas-samples/send_emails_rust_csharp): Send an email with Nylas REST APIs using Rust or C#!
 - [Send and Read Emails with PHP](https://github.com/nylas-samples/php_send_and_read_emails): Read or Send an email with Nylas REST APIs using PHP!
 
+## Account API samples
+
+**Node.js SDK**
+
+- [read-account](/code-samples/account/node/read-account/): Read account details using Nylas Node SDK!
```
Comment (relaxedtomato) [inline readme.md:89]:
```suggestion
- [node-read-account](/code-samples/account/node/read-account/): Read account details using Nylas Node SDK!
```

File: code-samples/account/node/read-account/.gitignore (hunk 1 above)
Comment (amit-ksh) [reply code-samples/account/node/read-account/.gitignore:15]: So what should I do is move the `.gitignore` to inside the `account/node/` directory or create a new  `.gitignore`

**Review by relaxedtomato — APPROVED** [review]
🚢

File: code-samples/account/node/read-account/.gitignore (hunk 1 above)
Comment (relaxedtomato) [reply code-samples/account/node/read-account/.gitignore:15]: Ah no - this is good to have. No changes.

**Comment by relaxedtomato** [comment]
Thanks @amit-ksh for contributing 🙌

**Pull request merged**